### PR TITLE
Unify headings titles and breadcrumbs for SDK 2.3.7 - Closes #480

### LINKED ISSUE #480: Unify headings, titles and breadcrumbs for SDK 2.3.7
- Same page title as the navigation title of the page for every page
- Update navigations to not use bold navigation links
reference/configuration reference: change title to be only "configuration"
- add banner images for Elements, Commander (like in the README files on GitHub)
- Use imperative form in page titles, e.g.: Config file handling --> Handle config files
- On the start page of every version, change the H0 title to Introduction, remove the subtitle Introduction if present
- Use sentence case in titles (small letters except first letter of first word, names, abbreviations,...)

### PR DESCRIPTION
**Pull request by chris529**
closes #480

### CODE REVIEW

File: modules/ROOT/pages/reference/lisk-commander/index.adoc [hunk 1]
```diff
@@ -1,4 +1,4 @@
-= Lisk Commander Documentation Overview
+= Lisk Commander documentation overview
```
Comment (Tschakki) [inline modules/ROOT/pages/reference/lisk-commander/index.adoc]: Change overview title to introduction:
Change to:
```suggestion
= Lisk Commander introduction
```

Also, the banner image seems to be missing still.

File: modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -1,4 +1,4 @@
-= Lisk SDK Overview
+= Introduction
```
Comment (Tschakki) [inline modules/ROOT/pages/index.adoc:1]:
```suggestion
= Lisk SDK Introduction
```

File: modules/ROOT/pages/reference/api.adoc [hunk 3]
```diff
@@ -1,24 +1,24 @@
-= Lisk API Documentation
+= Lisk API documentation
```
Comment (Tschakki) [inline modules/ROOT/pages/reference/api.adoc]: The content of this page is auto generated from the `swagger.yml` of the lisk-sdk.
To make the changes permanent, we need to change it over there.

File: modules/ROOT/pages/reference/config.adoc [hunk 4]
```diff
@@ -1,14 +1,14 @@
-= Lisk SDK Configuration Reference
+= Lisk SDK configuration reference
 Mona Bärenfänger <mona@lightcurve.io>
 :toc:
-:url_guide_config: basic-guides/configuration.adoc#modify_default_values
+:url_guide_config: basic-guides/configuration.adoc#modify defaultvalues
```
Comment (Tschakki) [inline modules/ROOT/pages/reference/config.adoc]: what happened here?

File: modules/ROOT/pages/reference/lisk-elements/index.adoc [hunk 5]
```diff
@@ -1,4 +1,4 @@
-= Lisk Elements Overview
+= Lisk Elements overview
```
Comment (Tschakki) [inline modules/ROOT/pages/reference/lisk-elements/index.adoc]:
```suggestion
= Lisk Elements introduction
```

Also, the banner image is missing.

File: modules/ROOT/pages/reference/lisk-framework/index.adoc [hunk 6]
```diff
@@ -1,4 +1,4 @@
-= Lisk Framework Overview
+= Lisk framework overview
```
Comment (Tschakki) [inline modules/ROOT/pages/reference/lisk-framework/index.adoc]:
```suggestion
= Lisk Framework introduction
```

Lisk Framework is the product name, and therefore capitalized.

Please fix also all other occurences of this on this page.

File: modules/ROOT/pages/reference/lisk-framework/index.adoc [hunk 7]
```diff
@@ -36,22 +36,22 @@ image:diagram_framework.png[Logo]
...
 The modules are individual building blocks for Lisk Core.
 
-==== Core Modules
+==== Core modules
 
 The core modules are shipped along with the Lisk Core distribution itself.
 These modules constitute the minimum requirements to run a functional Lisk Core instance.
 
-===== List of Core Modules
+===== List of Core modules
```
Comment (Tschakki) [inline modules/ROOT/pages/reference/lisk-framework/index.adoc:44]:
```suggestion
===== List of core modules
```

File: modules/ROOT/pages/tutorials/cashback.adoc [hunk 8]
```diff
@@ -22,17 +22,17 @@ Mona Bärenfänger <mona@lightcurve.io>
 A simple application which rewards the users for sending tokens.
 The cashback reward tokens are freshly minted, and in turn increase the absolute amount of tokens within the network.
 
-The Cashback Application implementation steps are shown below:
+The cashback application implementation steps are described below:
```
Comment (Tschakki) [inline modules/ROOT/pages/tutorials/cashback.adoc]: Imho, "Cashback" is the name of the app and therefore should bbe capitalized.

```suggestion
The Cashback application implementation steps are described below:
```

File: modules/ROOT/pages/setup.adoc [hunk 9]
```diff
@@ -1,4 +1,4 @@
-= SDK Setup
+= SDK setup
```
Comment (Tschakki) [inline modules/ROOT/pages/setup.adoc]: Please ensure that all headings of the pages are identical with the links in the navigation (`nav.adoc`).
In this case, it should be
 ```suggestion
= Prerequisites
```

File: modules/ROOT/pages/tutorials/transport0.adoc [hunk 10]
```diff
@@ -27,7 +27,7 @@ This document will explain how to prepare your computer for the workshop.
 * Jumper cables.
 * USB <-> Micro-USB cable.
 
-=== Software Pre-requisites
+=== Software pre-requisites
```
Comment (Tschakki) [inline modules/ROOT/pages/tutorials/transport0.adoc]:
```suggestion
=== Software prerequisites
```

**Review by Tschakki — CHANGES_REQUESTED** [review]
Please see my comments!

File: modules/ROOT/pages/reference/config.adoc (hunk 4 above)
Comment (chris529) [reply modules/ROOT/pages/reference/config.adoc]: Good question...Am not sure, but I have fixed it.. (see below)

:toc:
:url_guide_config: basic-guides/configuration.adoc##modify_default_values

File: modules/ROOT/pages/reference/lisk-elements/index.adoc (hunk 5 above)
Comment (chris529) [reply modules/ROOT/pages/reference/lisk-elements/index.adoc]: OK, I have added the banner...I am not sure this is the correct one, so can you please check it Mona..Thanks :-)

File: modules/ROOT/pages/reference/lisk-framework/index.adoc (hunk 6 above)
Comment (chris529) [reply modules/ROOT/pages/reference/lisk-framework/index.adoc]: OK, its now done :-)

File: modules/ROOT/pages/setup.adoc (hunk 9 above)
Comment (chris529) [reply modules/ROOT/pages/setup.adoc]: OK it should now be fine..

File: modules/ROOT/pages/reference/api.adoc (hunk 3 above)
Comment (chris529) [reply modules/ROOT/pages/reference/api.adoc]: OK,  if this is auto generated do I need to make any changes here..?

Comment (Tschakki) [reply modules/ROOT/pages/reference/api.adoc]: We can leave the existing changes for now, but they will be overwritten the next time we generate updated Lisk SDK API docs.

File: modules/ROOT/pages/reference/config.adoc [hunk 11]
```diff
@@ -1,14 +1,14 @@
-= Lisk SDK Configuration Reference
+= Lisk SDK configuration reference
 Mona Bärenfänger <mona@lightcurve.io>
 :toc:
-:url_guide_config: basic-guides/configuration.adoc#modify_default_values
+:url_guide_config: basic-guides/configuration.adoc##modify_default_values
```
Comment (Tschakki) [inline modules/ROOT/pages/reference/config.adoc]:
```suggestion
:url_guide_config: basic-guides/configuration.adoc#modify_default_values
```

File: modules/ROOT/pages/reference/lisk-elements/index.adoc [hunk 12]
```diff
@@ -1,26 +1,28 @@
...
 Mona Bärenfänger <mona@lightcurve.io>
 :description: The Lisk Elements overview provides an introduction to the Lisk Elements library and the different sub-packages. The overall basic usage and how to upgrade with NPM is described here..
 :toc:
-:imagesdir: ../..assets/images
+:imagesdir: ../../../assets/images
 :url_nodejs: https://nodejs.org/
 :url_nodejs_download: https://nodejs.org/en/download/
 :url_elements_packages: reference/lisk-elements/packages.adoc
 
+image::lisk_docu_blog_elements.png[Logo]
```
Comment (Tschakki) [inline modules/ROOT/pages/reference/lisk-elements/index.adoc]: I think it is outdated.
Use the same banner as in the lisk-sdk repo: https://github.com/LiskHQ/lisk-sdk/tree/development/elements

File: modules/ROOT/pages/reference/lisk-commander/index.adoc [hunk 13]
```diff
@@ -1,15 +1,17 @@
...
-:imagesdir: ../assets/images
+:imagesdir: ../../../assets/images
 
 :url_commander_commands: reference/lisk-commander/user-guide/commands.adoc
 :url_commander_user_guide: reference/lisk-commander/user-guide.adoc
 :url_nodejs_download: https://nodejs.org/en/download/
 :url_nodejs: https://nodejs.org/
 
-Lisk Commander is a Command Line Interface tool designed to be a low entry point and the most convenient tool for developers who are looking for an efficient way of coding.
+image::diagram_sdk.png[Logo]
```
Comment (Tschakki) [inline modules/ROOT/pages/reference/lisk-commander/index.adoc]: Use same banner as here: https://github.com/LiskHQ/lisk-sdk/tree/development/commander

File: modules/ROOT/pages/reference/lisk-framework/index.adoc (hunk 6 above)
Comment (Tschakki) [reply modules/ROOT/pages/reference/lisk-framework/index.adoc]: Looks like the title is still wrong. Please simply apply my suggestion.

File: modules/ROOT/pages/tutorials/transport1.adoc [hunk 14]
```diff
@@ -1,4 +1,4 @@
-= Supply Chain Tutorial Part 1: Track a packet on the blockchain
+= Supply chain tutorial part 1: Track a packet on the blockchain
```
Comment (Tschakki) [inline modules/ROOT/pages/tutorials/transport1.adoc]: This title does not match the link in the nav. Please ensure for all other titles that they match the nav link, there are probably more to fix.

**Review by Tschakki — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/basic-guides/configuration.adoc [hunk 15]
```diff
@@ -1,4 +1,4 @@
-= Lisk SDK Configuration
+= Lisk SDK configuration
```
Comment (chris529) [inline modules/ROOT/pages/basic-guides/configuration.adoc]: Done.

File: modules/ROOT/pages/basic-guides/customize.adoc [hunk 16]
```diff
@@ -50,7 +50,7 @@ The BaseTransaction class is the interface that all other Transaction Types need
 
 TIP: {url_github_base_transaction}[See the original file on Github].
 
-=== Required Properties
+=== Required properties
```
Comment (chris529) [inline modules/ROOT/pages/basic-guides/customize.adoc:53]: Done.

File: modules/ROOT/pages/basic-guides/customize.adoc [hunk 17]
```diff
@@ -62,7 +62,7 @@ static TYPE: number
 The hallmark of a transaction.
 Set this constant to any number, except `0-9`, which are reserved for the default transactions.
 
-=== Required Methods
+=== Required methods
```
Comment (chris529) [inline modules/ROOT/pages/basic-guides/customize.adoc:65]: Done.

File: modules/ROOT/pages/basic-guides/customize.adoc [hunk 18]
```diff
@@ -109,7 +109,7 @@ undoAsset(store: StateStore): ReadonlyArray<TransactionError>
 The inversion of the `applyAsset` method.
 Undoes all of the changes to the accounts applied by the `applyAsset` step.
 
-=== Additional Methods
+=== Additional methods
```
Comment (chris529) [inline modules/ROOT/pages/basic-guides/customize.adoc:112]: Done

**Review by chris529 — COMMENTED** [review]
Looks good..

File: modules/ROOT/pages/reference/lisk-elements/index.adoc (hunk 12 above)
Comment (chris529) [reply modules/ROOT/pages/reference/lisk-elements/index.adoc]: OK, it is now done..

File: modules/ROOT/pages/reference/lisk-commander/index.adoc (hunk 13 above)
Comment (chris529) [reply modules/ROOT/pages/reference/lisk-commander/index.adoc]: OK, it is also now done..

File: modules/ROOT/pages/tutorials/transport1.adoc (hunk 14 above)
Comment (chris529) [reply modules/ROOT/pages/tutorials/transport1.adoc]: should be ok now..

File: modules/ROOT/pages/reference/lisk-elements/index.adoc [hunk 19]
```diff
@@ -1,26 +1,28 @@
...
 Mona Bärenfänger <mona@lightcurve.io>
 :description: The Lisk Elements overview provides an introduction to the Lisk Elements library and the different sub-packages. The overall basic usage and how to upgrade with NPM is described here..
 :toc:
-:imagesdir: ../..assets/images
+:imagesdir: ../../../assets/images
 :url_nodejs: https://nodejs.org/
 :url_nodejs_download: https://nodejs.org/en/download/
 :url_elements_packages: reference/lisk-elements/packages.adoc
 
+image::banner_elements.png[Logo]
```
Comment (Tschakki) [inline modules/ROOT/pages/reference/lisk-elements/index.adoc:10]: The image is not displayed, please check

File: modules/ROOT/pages/reference/lisk-commander/index.adoc [hunk 20]
```diff
@@ -1,15 +1,17 @@
-= Lisk Commander Documentation Overview
+= Lisk Commander
 Mona Bärenfänger <mona@lightcurve.io>
 :description: The Lisk Commander overview provides an introduction to the Lisk Commander CLI tool its setup and usage.
 :toc:
-:imagesdir: ../assets/images
+:imagesdir: ../../../assets/images
```
Comment (Tschakki) [inline modules/ROOT/pages/reference/lisk-commander/index.adoc:5]: Image is not displayed

**Review by Tschakki — CHANGES_REQUESTED** [review]
Looks good, but the banner images are missing / not displayed, please check again!

File: modules/ROOT/pages/reference/lisk-elements/index.adoc (hunk 19 above)
Comment (chris529) [reply modules/ROOT/pages/reference/lisk-elements/index.adoc:10]: OK, it should now work :-)

File: modules/ROOT/pages/reference/lisk-commander/index.adoc (hunk 20 above)
Comment (chris529) [reply modules/ROOT/pages/reference/lisk-commander/index.adoc:5]: OK, it should now work :-)

File: modules/ROOT/nav.adoc [hunk 21]
```diff
@@ -1,18 +1,20 @@
 * xref:setup.adoc[Prerequisites]
 * xref:getting-started.adoc[Getting started]
+* xref:index.adoc[Introduction]
 * *Basic Guides*
```
Comment (Tschakki) [inline modules/ROOT/nav.adoc]: Please remove bold styling in all navigation items

**Review by Tschakki — CHANGES_REQUESTED** [review]
Nearly done, only some nav links are still bold, please fix this :)

File: modules/ROOT/nav.adoc (hunk 21 above)
Comment (chris529) [reply modules/ROOT/nav.adoc]: oops..yes I seem to have missed this. It should be fixed now ;-)

**Review by Tschakki — APPROVED** [review]
Thank you!

**Pull request merged**